documentation/4.0/content/managing-domains/domain-resource.md

```diff
@@ -206,8 +206,15 @@ The elements `serverStartPolicy`, `serverPod` and `serverService` are repeated u
 Elements related to the customization of liveness and readiness probes:
 * See [Liveness probe customization]({{< relref "/managing-domains/domain-lifecycle/liveness-readiness-probe-customization#liveness-probe-customization" >}}) for details about the elements related to liveness probe customization and [Readiness probe customization]({{< relref "/managing-domains/domain-lifecycle/liveness-readiness-probe-customization#readiness-probe-customization" >}}) for details about the elements related to readiness probe customization.
 
+Cluster spec elements:
+
+For a complete list of the Cluster spec elements, see [Cluster Spec](https://github.com/oracle/weblogic-kubernetes-operator/blob/{{< latestMinorVersion >}}/documentation/domains/Cluster.md#cluster-spec). The following paragraph has additional context information for the `clusterServer.sessionAffinity` element.
```

Comment:
The following paragraph has additional context information for the `clusterServer.sessionAffinity` element. -> re-write: Additional information for the `clusterServer.sessionAffinity` element:

Reply:
Changed.